src/open_inwoner/openklant/views/contactform.py

```diff
@@ -44,7 +52,8 @@ def get_initial(self):
                     "first_name": self.request.user.first_name,
                     "infix": self.request.user.infix,
                     "last_name": self.request.user.last_name,
-                    "email": self.request.user.email,
+                    # we need to use get_contact_email() because we use dummy email for BSN users
```

Comment:
This is a bit confusing for me!I thought we use dummy emails only for the registration. If the digid user logs in it's necessary to fill in the real email and so when accessing the form it will be available. Am I wrong?

Reply:
This is actually the only comment that Vasileios would like to see clarified, the other are trivial

Reply:
No you are right, I got confused by the contacts feature. I changed it to use normal .email